README.md

```diff
@@ -22,11 +22,12 @@ rules:
   from: 127.0.0.1:6443
   to:
     - 192.168.1.19:6443
-    - 192.168.1.21:6443
+    - 192.168.1.1@192.168.1.21:6443
```

Comment:
I don't think this makes sense.

What are you trying to achieve here?

Reply:
I have never seen this in cloud native software, I haven't heard why it would be absolutely needed for mixctl yet.